.github/workflows/enable-pack-release.yaml

```diff
@@ -0,0 +1,116 @@
+concurrency:
+  group: enable-pack-release-${{ github.ref }}
+  # Do not use cancel-in-progress! Or we might create a scenario where all workflows get canceled before they complete.
+
+permissions:
+  contents: read # Need to clone repositories
+
+jobs:
+  enable_pack_release_workflow:
+    runs-on: ubuntu-20.04
+    # When parent workflow is named "Enable Pack Release Workflow" this shows up as:
+    #   "Enable Pack Release Workflow / Enable Pack Release Workflow"
+    name: Enable Pack Release Workflow
+
+    steps:
+
+    - name: Add checkout paths to env context
+      shell: bash
+      run: |
+        echo "CI_DIR=${{ github.workspace }}/ci" >> ${GITHUB_ENV}
+        echo "TOOLS_DIR=${{ github.workspace }}/exchange-tools" >> ${GITHUB_ENV}
+        echo "PACKS_PATH=${{ github.workspace }}/packs" >> ${GITHUB_ENV}
+
+    - name: Checkout ci repo
+      uses: actions/checkout@v2
+      with:
+        repository: StackStorm-Exchange/ci
+        ref: ${{ env.ci_branch }}
+        path: ci
+        fetch-depth: 1
+
+    - name: Checkout exchange-tools repo
+      uses: actions/checkout@v2
+      with:
+        repository: StackStorm-Exchange/exchange-tools
+        ref: ${{ env.exchange_tools_branch }}
+        path: exchange-tools
+        fetch-depth: 1
+
+    - name: Checkout Pack Repos and enable their Release Workflows
+      shell: bash
+      env:
+        PACKS_ORG: ${{ env.packs_org }}
+        PACKS_PREFIX: ${{ env.pack_repo_prefix }}
+      run: |
+        mkdir -p ${PACKS_PATH}
+        cd ${PACKS_PATH}
+        source ${TOOLS_DIR}/functions.sh
+        for repo_name in $(_gh_list_repo_names ${PACKS_ORG} ${PACKS_PREFIX}); do
+          echo "::group::Clone ${PACKS_ORG}/${repo_name}"
+          gh repo clone "${PACKS_ORG}/${repo_name}"
+          if latestTag=$(git -C ${repo_name} describe --tags `git -C ${repo_name} rev-list --tags --max-count=1`); then
+            echo latestTag = $latestTag
+            git -C ${repo_name} checkout $latestTag -b latestTagBranch
+          fi
+          echo
+          echo "::endgroup::" # DELETED notices will not be folded to simplify scanning action output
+          if [[ -z "$latestTag" ]]; then
+            rm -rf ${repo_name}
+            echo "DELETED clone of ${PACKS_ORG}/${repo_name}: missing git tags"
+            echo "${PACKS_ORG}/${repo_name}" >> ${GITHUB_WORKSPACE}/bad_pack_repos
+          elif [[ ! -f "./${repo_name}/pack.yaml" ]]; then
+            rm -rf ${repo_name}
+            echo "DELETED clone of ${PACKS_ORG}/${repo_name}: missing pack.yaml"
+            echo "${PACKS_ORG}/${repo_name}" >> ${GITHUB_WORKSPACE}/bad_pack_repos
+          else
+            #
+            #  This is not a bad Pack, ensure the Release workflow is active
+            #
+            echo "::group::Ensure Release workflow for ${PACKS_ORG}/${repo_name} is active"
+            RELEASE_WORKFLOW="repos/${PACKS_ORG}/${repo_name}/actions/workflows/release.yaml"
```

Comment:
Just a note, it's the `CI` (`build_and_test.yam`l) workflow that needs to be re-activated instead of "Release".

![Screenshot 2022-12-17 at 14-53-05 Actions · StackStorm-Exchange_stackstorm-ansible](https://user-images.githubusercontent.com/1533818/208247836-f481bfd6-6c5d-4cce-9fbe-9ac6d7da4ec7.png)